nucleus/__init__.py

```diff
@@ -181,11 +181,13 @@ def list_models(self) -> List[Model]:
 
         return [
             Model(
-                model["id"],
-                model["name"],
-                model["ref_id"],
-                model["metadata"],
-                self,
+                model_id=model["id"],
+                name=model["name"],
+                reference_id=model["ref_id"],
+                metadata=None
+                if model["metadata"] == {}
```

Comment:
Consider using `model["metadata"] or None` to shorten this, but it might not necessarily be more readable ([SO](https://stackoverflow.com/a/38513904)).